app/services/AtsService.scala

```diff
@@ -139,6 +145,16 @@ trait AtsService {
           "taxYear" -> data.taxYear.toString,
           "time" -> new Date().toString
         ))
+      case _: Nino =>
+        val userType = if (AccountUtils.isPortalUser(request)) "non-transitioned" else "transitioned"
+        auditService.sendEvent(AuditTypes.Tx_SUCCEEDED, Map(
+          "userId" -> request.userId,
+          "nino" -> data.nino.get,
+          "userType" -> userType,
+          "taxYear" -> data.taxYear.toString,
+          "time" -> new Date().toString
+        ))
```

Comment:
Is this new functionality or has it been refactored to here?

Reply:
I  believe I just copied the utr case to ensure that we had audit events for PAYE too. I don't remember writing a test for this, and I haven't seen any audit events either because we've been running on localhost. This is a good candidate for a separate PR with unit tests, that we can merge into master on its own. As we've said in the past, the plan is to break this into small PR's that we can merge into  hmrc master one at a time.